### DISCUSSION TITLE
Is there a way to override defaults for Button objects?

### DISCUSSION THREAD

**Discussion by jeffwright13** [bczsalba/pytermgui#58] [Q&A] [answered]
I'm looking for a way to disable the default square-bracket delimiters for Button object titles. That is, instead of a Quit button being labeled `[ Quit ]`, I'd like it to just show up as `Quit`. 

I'd also like the button label text to be just plain black, not the vaguely-olive-drab/tan color it is now. 

I've been looking through the code and it's not immediately obvious to me how to do either of these tasks.

Any clues for me? :-)

**Comment by bczsalba** [comment, marked as answer]
The `[` `]` part can be achieved by the chars API:

```python3
Button.set_char("delimiter", [" ", " "])
```

And styling is done through the styling API:

```python3
Button.styles.label = "@black white"
Button.styles.highlight = "@white black"
```

The chars API will likely change at some point into something closer to the styles one, though the original API will probably remain for some uses.
